pkg/ip/parse_ip_net.go

```diff
@@ -0,0 +1,39 @@
+package ip
+
+import (
+	"net"
+	"strings"
+)
+
+func ParseIPNet(s string) *net.IPNet {
```

Comment:
Why not return errors from this function rather than just returning `nil` IPNet? Might be more useful for the validation if we have descriptive errors?

Reply:
Yes maybe, but I was sort of following the pattern of `net.ParseIP`.   There's really only one meaningful failure case here: "not recognized".  The best differentiation I can produce is "looks like IP address, but couldn't parse" or "looks like CIDR range, but couldn't parse".  I believe that the ParseCIDR produces only one error.

Reply:
Ok, let's leave it this way then